docs/user/production-considerations/production.asciidoc

```diff
@@ -16,6 +16,12 @@ separate from  your {es} data or master nodes. To distribute {kib}
 traffic across the nodes in your {es} cluster,
 you can configure {kib} to use a list of {es} hosts.
 
+[WARNING]
+====
+Running more than one {kib} version against the same {es} index is unsupported.
+Upgrading while older {kib} instances are running can cause data loss or upgrade failures.
+====
```

Comment:
Saying `against the same {es} index` is still a bit confusing, as with the [removal of legacy multitenancy](https://github.com/elastic/kibana/issues/82020) you can no longer run multiple different "Kibanas" on the same ES cluster: you can run multiple identical Kibana instances to load balance the traffic, but those instances cannot have data isolation, i.e. they will all behave as if they are the "same" Kibana

So we are sort of combining warnings about a few different things here:
1. Warning users that they cannot run more than one Kibana deployment against an ES cluster. (All instances are sharing the same data)
2. Warning users that when running multiple instances of Kibana for the purposes of load balancing, they cannot be different versions.
3. Warning users that when upgrading Kibana, they must not have any other Kibana instances still running. (If they are older versions, there is a risk of data loss)

With that in mind, I'd propose rewording this a bit, maybe something like:

> {kib} does not support rolling upgrades, and deploying mixed versions of {kib} can result in data loss or upgrade failures. Please shut down all instances of {kib} before performing an upgrade, and ensure all running {kib} instances have matching versions. 

LMK what you think.

Reply:
I think you suggestion is correct.